test/java/seedu/addressbook/data/AddressBookTest.java

```diff
@@ -48,25 +51,25 @@ public void setUp() throws Exception {
                                     new Phone("91235468", false),
                                     new Email("alice@nushackers.org", false),
                                     new Address("8 Computing Drive, Singapore", false),
-                                    new UniqueTagList(tagMathematician));
+                                    Collections.singleton(tagMathematician));
 
         bobChaplin     = new Person(new Name("Bob Chaplin"),
                                     new Phone("94321500", false),
                                     new Email("bob@nusgreyhats.org", false),
                                     new Address("9 Computing Drive", false),
-                                    new UniqueTagList(tagMathematician));
+                                    Collections.singleton(tagMathematician));
 
         charlieDouglas = new Person(new Name("Charlie Douglas"),
                                     new Phone("98751365", false),
                                     new Email("charlie@nusgdg.org", false),
                                     new Address("10 Science Drive", false),
-                                    new UniqueTagList(tagScientist));
+                                    Collections.singleton(tagScientist));
 
         davidElliot    = new Person(new Name("David Elliot"),
                                     new Phone("84512575", false),
                                     new Email("douglas@nuscomputing.com", false),
                                     new Address("11 Arts Link", false),
-                                    new UniqueTagList(tagEconomist, tagPrizeWinner));
+                                    new HashSet<>(Arrays.asList(tagEconomist, tagPrizeWinner)));
```

Comment:
[2/4] If want to make it immutable, `Set.of(...)` would probably be a better option.

Reply:
This is currently still in Java 8, so `Set.of(...)` is still not available yet 😞.

Reply:
oh... crap lol